Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/AddMoodFragment.java

```diff
@@ -269,4 +269,10 @@ public void onMoodUpdateSuccess(){
         getActivity().finish();
     }
 
+    /**
+     * Function will not be triggered
+     */
+    @Override
```

Comment:
Hmm, maybe to avoid having to have AddMoodFragment implement this function and not do anything, it might be better to make another interface for delete. So instead of using OnMoodUpdateListener in the MoodHistoryFragment, make another interface like OnMoodDeleteListener with the functions needed.